modules.py

```diff
@@ -219,18 +249,17 @@ def __init__(self, wavelength, array_size=101):
         self.name = 'Pupil'
 
     def _compute_pupil(self):
-        print 'Computing pupil...'
+        logging.info("Computing pupil...")
 
         aperture = Aperture(self.array_size)
         if glob.glob('aperture.fits'):
-            print "... using 'aperture.fits'..."
+            logging.info("... using 'aperture.fits'...")
```

Comment:
is this the name of the fits file always? I thought you were switching to a variable name.